packages/ui/src/pages/Settings/WatchedAccounts.tsx

```diff
@@ -0,0 +1,135 @@
+import { styled } from '@mui/material/styles'
+import { Box, Grid, IconButton, Paper } from '@mui/material'
+import { useWatchedAddresses } from '../../contexts/WatchedAddressesContext'
+import AccountDisplay from '../../components/AccountDisplay'
+import { HiOutlineXMark } from 'react-icons/hi2'
+import AccountSelection from '../../components/select/AccountSelection'
+
+interface Props {
+  className?: string
+}
+
+const WatchedAccounts = ({ className }: Props) => {
+  const { watchedAddresses, removeWatchedAccount, addWatchedAccount } = useWatchedAddresses()
+
+  return (
+    <>
+      <WatchAccountsHeaderStyled>Currently watched accounts:</WatchAccountsHeaderStyled>
+      <Grid
+        className={className}
+        container
+        spacing={2}
+      >
+        {watchedAddresses.length > 0 && (
+          <Grid
+            item
+            xs={12}
+            md={8}
+          >
+            <PaperStyled>
+              {watchedAddresses.map((address, index) => {
+                return (
+                  <AccountItemWrapperStyled
+                    key={address}
+                    data-cy="container-account-details"
+                  >
+                    <AccountCountStyled>{index + 1}</AccountCountStyled>
```

Comment:
Sorry that I didn't see this before in the design, but this is not needed, as it takes space and doesn't add value.
```suggestion
                    <AccountCountStyled>{index + 1}</AccountCountStyled>
```